src/promptflow/promptflow/azure/_cli/_pf_azure/_flow.py

```diff
@@ -16,13 +16,13 @@
     add_param_set,
```

Comment:
If these files are moved to `promptflow/azure`, i think we can remove the `_pf_azure` sub-folder then? like `promptflow/azure/_cli/_flow.py`

Reply:
looks good.